.gitlab/kitchen_testing/new-e2e_testing.yml

```diff
@@ -48,7 +42,7 @@
     EXTRA_PARAMS: --osversion $E2E_OSVERS --platform $E2E_PLATFORM --arch $E2E_ARCH --flavor $FLAVOR
   parallel:
     matrix:
-      - START_MAJOR_VERSION: [5, 6, 7]
+      - START_MAJOR_VERSION: [6]
         END_MAJOR_VERSION: [7]
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** I'm not sure about this but should this be changed as well?‏

Reply:
This seems to be a test of agent upgrade from 6 to 7, so I thought it was expected to keep it